python/ray/experimental/placement_group.py

```diff
@@ -0,0 +1,152 @@
+import copy
+import enum
+import random
+
+import ray
+
+LABEL_RESOURCE_AMOUNT = 2 ** 15
+
+
+@enum.unique
+class PlacementStrategy(enum.Enum):
+    PACK = 0
+    SPREAD = 1
```

Comment:
For Python we should just use `strategy="pack"` or `strategy="spread"` for simplicity, and raise ValueError if the argument is not one of these.

Reply:
LGTM. String is much simpler than enumeration.